decentraland/unity-renderer
# chore: removed scripts/prefabs related to unused MOTD

## What does this PR change?

Fix #1906
Removes unused stuff related to MOTD

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/remove-message-of-the-day
2. Verify that everything works fine

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
